package.json

```diff
@@ -131,7 +132,9 @@
     "prepare": "webpack --config webpack/publish.config.js --progress",
     "start": "node ./Parse-Dashboard/index.js",
     "madge:circular": "node_modules/.bin/madge ./src --circular",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run"
```

Comment:
Hi @visualfanatic, 
First of all thanks for this PR !  🙂 

I have set up Cypress in my company and we use it extensively to cover our apps , and our apps mainly use parse-server as the backend framework.

On my team monorepo, I made the choice to start with a completely native structure:
- Our backend is started by npm script natively, here we are also lucky we can here start parse server via CLI, thanks to the `parse-server` command, if we install parse-server as a devDep
- Also parse-server plays very well with mongodb runner (here "pretest" command: https://github.com/parse-community/parse-server/blob/43cc97d99fa7346739f9e376a22afe3d95439b4c/package.json#L123 )

Here i can suggest may be remove docker need for local development and directly work in native env with parse-server CLI and mongodb runner.

I can also suggest to add 2 commands here:
- "test:watch": which start parse+ mongo and then open cypress in UI mode
- "test": which start parse + mongo and then run cypress (headless)

Then a developer will be able to just `clone`, `npm i`  and `npm test:watch` without docker or manually launching/waiting services to be up.

I can also suggest here this popular package to ensure parse server startup before lauching cypress: https://www.npmjs.com/package/start-server-and-test

Reply:
With my team I am very sensitive to the developer experience, i think a native usage with a single command usage (all in one) could be more accessible for futur contributors and will facilitate new cypress tests contributions.

Reply:
I would not want to ditch the docker environment completely, it's just another option for people that prefer it. If we allow using a local backend for E2E tests we need to make sure all features required by tests are enabled and set up properly. It also changes mindset when you write a test, because you have to think about what other data there may be (or not) and how it can affect the test result. What if a flaky test overwrites some important data? I can think of a lot of similar concerns, that is why I would propose to allow both methods. Docker for those who prefer it and local server with a cautionary note about possible side effects. We could provide instructions for both methods with convenient all-in-one commands as you suggested.

@mtrezza @Moumouls what do you think?

Reply:
I'd suggest to focus on the native usage at first. Docker could be an additional env to set up and on GitHub it may slow down the CI significantly to the point where it's not efficient to run the CI for every PR commit. Maybe docker env could be a complementary PR later on when we have more insight into the behavior of native testing?

Reply:
@damianstasik sorry for the long wait time, it slipped through my notifications that you were waiting for a response here.